sklearn/tests/test_pipeline.py

```diff
@@ -5,22 +5,28 @@
 from scipy import sparse
 
 from sklearn.externals.six.moves import zip
-from sklearn.utils.testing import assert_raises
-from sklearn.utils.testing import assert_raises_regex
-from sklearn.utils.testing import assert_raise_message
-from sklearn.utils.testing import assert_equal
-from sklearn.utils.testing import assert_false
-from sklearn.utils.testing import assert_true
-from sklearn.utils.testing import assert_array_equal
-from sklearn.utils.testing import assert_array_almost_equal
-from sklearn.utils.testing import assert_warns_message
-from sklearn.utils.testing import assert_dict_equal
+from sklearn.utils.testing import (
```

Comment:
Some people like one, some the other. We've gone between them many times. The other one is more robust to merge conflicts, I think.